pyproject.toml

```diff
@@ -5,9 +5,11 @@ requires = [
     "setuptools>=38.6.0",
     "wheel",
     "Cython>=0.29.21,<3",  # Note: sync with setup.py
-    "numpy==1.17.3; python_version=='3.7'",
-    "numpy==1.18.3; python_version=='3.8'",
-    "numpy; python_version>='3.9'",
+    # Note: ARM64 needs newer numpy
+    "numpy==1.17.3; python_version=='3.7' and platform_machine!='arm64'",
+    "numpy==1.18.3; python_version=='3.8' and platform_machine!='arm64'",
+    "numpy>=1.19.3; python_version>='3.9' and platform_machine!='arm64'",
+    "numpy>=1.20.0; platform_machine=='arm64'"
```

Comment:
See https://github.com/scipy/scipy/blob/master/pyproject.toml. We might need something like that.